src/main/java/jenkins/plugins/slack/CommitInfoChoice.java

```diff
@@ -0,0 +1,61 @@
+package jenkins.plugins.slack;
+
+public enum CommitInfoChoice {
+    NONE("nothing about commits") {
```

Comment:
That's fine, like I said I'm unfamiliar. Probably ok to leave it as hardcoded for now.